docs/tests.md

```diff
@@ -0,0 +1,454 @@
+# Tests
+
+## Testing Suite Architecture Overview
+
+This section provides an overview of the testing architecture for our Python
+library, which employs two main testing approaches:
+
+- **Known Answer Tests (KATs):** These tests, standard in cryptography, verify
+  the correctness of our implementation by comparing outputs with known results
+  for specific inputs. This approach ensures that cryptographic primitives and
+  algorithms are implemented correctly.
```

Comment:
**Known Answer Tests (KATs):** These tests verify the accuracy of the estimates made by the `CryptographicEstimators` library. Here one compares the estimates by `CryptographicEstimators` with the one of existing external estimators which are widely accepted as correct by the cryptographic community.

Reply:
Applied.